### ISSUE TITLE
[Suggestion] Armor & Tools for each individual Mob

### ISSUE THREAD

**Issue by Xaikii** [SilentChaos512/ScalingHealth#93]
Currently I've seen that it's only possible to set in general to give them Armor in generally.
I think it'd be nice if we'd be able to set for every mob possible Armor they could get depending on their Difficulty.

Example:
modid:name; difficulty; armor_piece; unlocalized_name; weight
minecraft:zombie; 250; chest; minecraft:diamond_chestplate; 100

**Comment by SilentChaos512** [comment]
I'll consider it. That would probably be a major rewrite of how blights are given armor.

**Comment by Xaikii** [comment]
Yeah I thought so ^^ '
And also I didnt only mean blights with that, basically why I am requesting this is to balance everything a bit better. Looking through all the Mods I founded there is basically none able to balance everything. You'd have to decide Harder Early or Late Game, and since Scaling Health has already a bunch of balance features for mobs, this'd be thing to increase balance even more. 

Btw while looking through the configs I noticed it is only possible to set additional health for non-blighted mobs, is intended/should I regen the config or is it just missing?
